postgres/schema.sql

```diff
@@ -89,6 +89,8 @@ CREATE TABLE events (
 		OR thumbnail_mode = 'NONE'
 		OR thumbnail_last_written IS NOT NULL
 	),
+	thumbnail_crop INTEGER[], -- left, upper, right, and lower pixel coordinates to crop the selected frame
+	thumbnail_location INTEGER[], -- left, top, right, bottom pixel coordinates to position the cropped frame
```

Comment:
It would be good to verify that these values are only ever 4 items, if they're set at all. eg. `CHECK cardinality(thumbnail_crop) = 4 OR thumbnail_crop IS NULL`